apps/site/src/_pages/docs/2_working-with-types.mdx

```diff
@@ -17,7 +17,7 @@ The system by which types are defined is known as the 'type system'.
 The Block Protocol type system is designed so that types are composed of re-usable, sharable parts, which can be
 referenced by other users and by other types.
 
-The type system consists of these primitive parts:
```

Comment:
Trying to avoid saying "primitive" because we have "primitive data types" and sometimes refer to the top-level link type as the "primitive link type"